src/main/resources/static/homepage.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
```

Comment:
Not sure what you want to do with this, but the `homepage.html` Thymeleaf template should reside in `resources/templates`, not in the `static` folder.
That's why you won't need the `spring.resources.static-locations=classpath:/static/` property in your `application.properties` file.

Reply:
deleted `spring.resources.static-locations=classpath:/static/` from `application.properties` and created controller in order to access homepage